src/hooks/useLogoutTimer.ts

```diff
@@ -67,6 +69,10 @@ export const useLogoutTimer = (callback: TCommonFunction) => {
 	useEffect(() => {
 		startTimer();
 
+		if (timerRef.current === timeBeforeLogout) {
+			setIsOpenInactivityLogoutModal(true);
+		}
+
```

Comment:
this condition is unattainable. better create new timer with 14 min duration and open a modal window after the timer expires